Schools/_posts/2019-02-02-Trans-Siberian-School.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 2nd Trans-Siberian School on High Energy Physics
+source: http://hep2019.tpu.ru/webcenter/portal/transib2019/page11?_adf.ctrl-state=3sr6xde5g_57
```

Comment:
Just one last thing - that URL looks very odd. Surely the POST information shouldn't be there. i.e., it should just be

<http://hep2019.tpu.ru/webcenter/portal/transib2019/page11>

Reply:
OK, it was such a small change I just did it myself...